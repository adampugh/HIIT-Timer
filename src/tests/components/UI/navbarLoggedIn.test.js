import React from "react";
import { shallow } from "enzyme"; 

import Navbar from "../../../components/UI/navbarLoggedIn";

test("should render navbarLoggedIn correctly", () => {
    const wrapper = shallow(<Navbar />);
    expect(wrapper).toMatchSnapshot();
});