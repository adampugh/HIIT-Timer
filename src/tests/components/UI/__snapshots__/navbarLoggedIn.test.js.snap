// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render navbarLoggedIn correctly 1`] = `
<nav
  className="navbar navbar--loggedIn"
>
  <div
    className="container"
  >
    <div
      className="navbar__items"
    >
      <h2>
        HIIT Timer
      </h2>
      <div
        className="navbar__links"
      >
        <NavLink
          activeClassName="active"
          ariaCurrent="true"
          to="/workouts"
        >
          <h3>
            Workouts
          </h3>
        </NavLink>
        <NavLink
          activeClassName="active"
          ariaCurrent="true"
          to="/"
        >
          <h3>
            Log Out
          </h3>
        </NavLink>
      </div>
    </div>
  </div>
</nav>
`;
